packages/bpk-component-horizontal-nav/src/BpkHorizontalNav.js

```diff
@@ -145,29 +153,40 @@ class BpkHorizontalNav extends Component<Props> {
       leadingScrollIndicatorClassName,
       trailingScrollIndicatorClassName,
       showUnderline,
+      type,
       ...rest
     } = this.props;
 
     const classNames = getClassName(
       'bpk-horizontal-nav',
-      showUnderline && 'bpk-horizontal-nav--show-underline',
+      showUnderline &&
+        type === HORIZONTAL_NAV_TYPES.default &&
+        'bpk-horizontal-nav--show-default-underline',
```

Comment:
It might be neater to do something like 
```
const underlineClassNames = {
  default: 'bpk-horizontal-nav--show-default-underline',
  light: 'bpk-horizontal-nav--show-light-underline',
}
...
showUnderline && underlineClassNames[type]
```

Reply:
Good idea, I thought of an even shorter way:

```
    const classNames = getClassName(
      'bpk-horizontal-nav',
      showUnderline && `bpk-horizontal-nav--show-${type}-underline`,
      className,
    );
```